src/patternfly/components/Table/examples/Table.md

```diff
@@ -1446,7 +1456,7 @@ Note: To apply padding to `.pf-c-table__expandable-row`, wrap the content in `.p
 {{/table}}
 ```
 
-### Expandable, hoverable, and selectable
+### Expandable, hoverable, and checkbox selectable example
```

Comment:
in this section, it looks like "selectable" revers to being able to the hoverable/selectable row feature, not the checkbox selectable. It's a bit confusing though because these examples also include checkboxes. Also compared to the react examples since we don't have an example of this version of a selectable table yet (https://github.com/patternfly/patternfly-react/issues/5444), and "selectable" refers to the check and radio selectable variations.

```suggestion
### Expandable, hoverable, and selectable example
```